components/kyma-environment-broker/cmd/broker/suite_test.go

```diff
@@ -432,11 +451,14 @@ func (s *OrchestrationSuite) WaitForOrchestrationState(orchestrationID string, s
 }
 
 func (s *OrchestrationSuite) AssertRuntimeUpgraded(runtimeID string, version string) {
-	assert.True(s.t, s.provisionerClient.IsRuntimeUpgraded(runtimeID, version), "The runtime %s expected to be upgraded", runtimeID)
+	c, _ := s.reconcilerClient.LastClusterConfig(runtimeID)
+	assert.Equal(s.t, version, c.KymaConfig.Version, "The runtime %s expected to be upgraded", runtimeID)
 }
 
 func (s *OrchestrationSuite) AssertRuntimeNotUpgraded(runtimeID string) {
-	assert.False(s.t, s.provisionerClient.IsRuntimeUpgraded(runtimeID, ""), "The runtime %s expected to be not upgraded", runtimeID)
+	_, err := s.reconcilerClient.LastClusterConfig(runtimeID)
```

Comment:
nit: just verifying this change is intentional, the previously called `s.provisionerClient.IsRuntimeUpgraded(runtimeID, "")` seems to have been designed for this, is it no longer good to use?

Reply:
in Kyma 2, the runtime upgrade we expect reconciler call instead of provisioner

Reply:
ah, right, of course :)